man/systemd.unit.xml

```diff
@@ -1412,6 +1417,10 @@
     <citerefentry><refentrytitle>systemd.automount</refentrytitle><manvolnum>5</manvolnum></citerefentry>
     for details. <varname>TriggersBy=</varname> is created implicitly on the
     triggered unit.</para>
+
+    <para>Note: <varname>Following=</varname> is used to group device aliases and points to the
+    "primary" device unit that systemd is using to track device state, usually corresponding to a
+    sysfs path. It does not show up in the "target" unit.</para>
```

Comment:
"sysfs path" sounds unnecessarily hackertalky... maybe say `device path in the <filename>/sys/</filename>` file system or so?